apps/calendar/js/performance.js

```diff
@@ -13,13 +13,24 @@ exports._isMonthReady = false;
 exports._isVisuallyActive = false;
 exports._isPendingReady = false;
 
+// TODO: It would be nice if this had an events interface so I could
+//     simply do performance.once('moz-app-loaded', () => ...) and
+//     I would be called immediately if we were already loaded or
+//     when we're loaded otherwise.
+var dispatched = {};
```

Comment:
I guess we will need to start using signals in the future :D many cases where it would simplify things.